modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/AllowUsersToEditSchemaProperties.testcase

```diff
@@ -238,7 +238,7 @@ definition {
 		}
 
 		task ("Then all fields of APISort object definition are present") {
-			var fieldNameList = "Author,Create Date,External Reference Code,ID,Modified Date,Status,OData Sort,UniversityAPISorts";
```

Comment:
I think this fix is already included in this PR: https://github.com/brianchandotcom/liferay-portal/pull/156050

Reply:
I honestly don't understand how it got here, I created the branch from master and I don't see it as a change in this commit locally 🤯

Reply:
yes, it is!

Reply:
@ccorreagg updated